README.md

```diff
@@ -293,6 +293,11 @@ export class Bus extends Car {
 }
 ```
 
+### Container clearing

```

Comment:
I see no harm in documenting public API. Extraneous documentation is always better, then the missing one. And this is in the end of the document, so we are not forcing anyone to read it.